cmd/endpoint.go

```diff
@@ -229,6 +227,14 @@ func NewEndpointList(args ...string) (endpoints EndpointList, err error) {
 
 	sort.Sort(endpoints)
 
+	if len(endpoints) > 1 {
```

Comment:
In what case, `len(endpoints)` <= 1 ?

Reply:
Why are you calling `NewEndpointList()` with single arg?  If so, it fails with correct message currently.  On what purpose you want to change?

Reply:
This is an older code to combine reading endpoints from a file,  it is being changed to a different approach. But in any case i would suggest not review endpoint.go for now since it has to handle a new ellipses syntax and things will change.